src/less/px.less

```diff
@@ -1,8 +1,8 @@
 /* Keyframes loaded early to please IE */
-@import "_keyframes";
+// @import "_keyframes";
```

Comment:
Are we going to use this at some point?

Reply:
Also not sure, was going to check with Erik due to the comment above.

Reply:
?

Reply:
No, was waiting for him to come into work, but i'll just message him about it on slack.

Reply:
It was left over from the old Design Guide, i'll test the animation on explorer.

Reply:
Works fine.